### ISSUE TITLE
Feature Request: Incorporating programmatic focus while using JAWS on browse mode using arrow keys

### ISSUE THREAD

**Issue by prashanthraj88** [FreedomScientific/VFO-standards-support#349]
### Summary

Feature Request: 

JAWS does not have a programmatic way of focusing an item in virtual cursor/browse mode when using the ARROW keys, unlike NVDA which does this by default. We see merit of this feature with NVDA. Hereby, submitting a feature request to incorporate programmatic focus while using JAWS on browse mode.

#### JAWS version and build number
JAWS 2019.1907.31 ILM
 
#### Operating System and version
Windows 10

#### Browser and version:
Firefox v71.0

**Comment by JAWS-test** [comment]
I'm not sure what you mean...

* JAWS can automatically switch to forms mode at input fields when you read with the arrow keys.
* When reading with arrow keys, Enter can be used to switch to forms mode for interactive elements, or the elements (such as links and buttons) can be operated directly.
* In addition, different access keys can be used to drag the various cursor types from JAWS to each other.

**Comment by turjmner8** [comment]
At least for my cases, the feature that was originally requested here would be very useful for 'grid' role elements. By using 'Enter' to switch to forms mode, a 'click' event is simulated rather than a programatic focus shift. In something like a grid that supports selection and other interactions, instead of focus alone being shifted to the given cell when this occurs, a selection may be made that was not intended by the user.

There is basically no way to start interacting with contents of cells in a given row without triggering a click on that cell / row. For example, if I use 'T' to jump to a grid that supports row selections, use table commands (ctrl+alt+arrows) to find the grid cell I want to interact with, and then try to shift focus to those cell contents without also selecting the row, there is not much I can do as a user. I would have to go somewhere else on the page, enter forms mode there, and then use Tab to get to the grid and navigate to the cell using the grid implementation itself.

NVDA allows for pressing Enter or Space to simulate a click on the current element when using browse mode similar to how JAWS currently works, but it also allows for a programatic focus shift without the added 'action' gesture when using Insert + Enter which I don't think JAWS supports.
